state/sqlserver/sqlserver.go

```diff
@@ -65,22 +68,26 @@ const (
 const (
 	connectionStringKey  = "connectionString"
 	tableNameKey         = "tableName"
+	metadataTableNameKey = "metadataTableName"
 	schemaKey            = "schema"
 	keyTypeKey           = "keyType"
 	keyLengthKey         = "keyLength"
 	indexedPropertiesKey = "indexedProperties"
 	keyColumnName        = "Key"
 	rowVersionColumnName = "RowVersion"
 	databaseNameKey      = "databaseName"
-
-	defaultKeyLength = 200
-	defaultSchema    = "dbo"
-	defaultDatabase  = "dapr"
-	defaultTable     = "state"
+	cleanupIntervalKey   = "cleanupIntervalInSeconds"
+
+	defaultKeyLength       = 200
+	defaultSchema          = "dbo"
+	defaultDatabase        = "dapr"
+	defaultTable           = "state"
+	defaultMetaTable       = "dapr_metadata"
+	defaultCleanupInterval = time.Hour
 )
 
-// NewSQLServerStateStore creates a new instance of a Sql Server transaction store.
-func NewSQLServerStateStore(logger logger.Logger) state.Store {
+// New creates a new instance of a Sql Server transaction store.
```

Comment:
```suggestion
// New creates a new instance of a SQL Server transaction store.
```

I'm a former Product Marketing Manager, please forgive me :)